### PR TITLE
Added Unique Subreddits feature for Every Guild

### PR DESCRIPTION
**Pull request by Rajrup911**
Added a  `.json` file where the `key` is the `guild id` in string and `value` will be an array of subreddit names to be used in that guild.

Now every server will have their own subreddit list from where they can choose...

### CODE REVIEW

File: src/cogs/redditAPI.py [hunk 1]
```diff
@@ -171,21 +202,26 @@ async def sublist(self, ctx):
...
-                f5.write("\n")
-            f5.write(s)
-            f5.close()
-
-        memes_list = []
-        f = open('./cogs/subreddit.txt', 'r')
-        memes_list = f.readlines()
-        f.close()
+
+        subsi_list = []
```
Comment (sdk2k01) [inline src/cogs/redditAPI.py:206]: Inappropriate naming :
Naming the list as `subsi_list` is quite ambiguous and doesn't clearly indicate the reasoning for such a name. Change all subsequent occurences of the same into some appropriate / self-explanatory name

File: src/cogs/redditAPI.py [hunk 2]
```diff
@@ -171,21 +202,26 @@ async def sublist(self, ctx):
...
+        guild = ctx.message.guild.id
+
+        with open ('./cogs/subred.json', 'r') as f:
+            subs = json.load(f)
+        
+        memes_list = subs[str(guild)]
+        bech = memes_list[-1] + '\n'
```
Comment (sdk2k01) [inline src/cogs/redditAPI.py:213]: another instance of use of unexplained naming of variable `bech`

**Review by sdk2k01 — COMMENTED** [review]
Please take a look at suggestions on variable names.

File: src/cogs/redditAPI.py (hunk 2 above)
Comment (Rajrup911) [reply src/cogs/redditAPI.py]: updated ..!!

File: src/cogs/redditAPI.py (hunk 1 above)
Comment (Rajrup911) [reply src/cogs/redditAPI.py]: updated ..!!

**Review by sdk2k01 — APPROVED** [review]
LGTM. Ready To Merge.

**Comment by sdk2k01** [comment]
Epiko

**Pull request merged**